src/main/scopes/wc/wc-scope.ts

```diff
@@ -226,10 +228,12 @@ export class WcScope extends Scope {
 
         for (const importPath of indexImports) {
           const segments = importPath.split('/')
-          const componentName = segments[segments.length - 1]?.replace(/\.js$/, '') ?? ''
+          const fileName = segments[segments.length - 1]?.replace(/\.js$/, '') ?? ''
+          const prefix = getWcPrefix(importPath)
+          const componentName = prefix === '' ? `${prefix}-${fileName}` : fileName
```

Comment:
```suggestion
          const componentName = prefix !== '' ? `${prefix}-${fileName}` : fileName
```
I think the logic here was flipped